mypy/semanal.py

```diff
@@ -2674,8 +2674,9 @@ def visit_member_expr(self, expr: MemberExpr) -> None:
                 # one type checker run. If we reported errors here,
                 # the build would terminate after semantic analysis
                 # and we wouldn't be able to report any type errors.
-                full_name = '%s.%s' % (file.fullname() if file is not None else None, expr.name)
-                mod_name = " '%s'" % file.fullname() if file is not None else ''
+                full_name = '%s.%s' % (file.fullname() if file is not None  # type: ignore
+                                       else None, expr.name)
+                mod_name = " '%s'" % file.fullname() if file is not None else ''  # type: ignore
```

Comment:
No, it's because of #2999. A bug fix in this commit triggered that issue. @JukkaL suggested to replace `#  type: ignore` with `cast` to make it safer, so I'll do that.